src/tutorial/runtime/introduction.rst

```diff
@@ -291,60 +275,161 @@ Where Do All The Files Go?
+
+   .. versionchanged:: 8.0.0
+
+      ``cylc install`` is a new feature of Cylc 8.
+
+   To allow you to separate the development and running of workflows
+   Cylc provides a :term:`cylc install <install>` command.
+
+For a workflow developed in
+
+- ``~/cylc-src/my_workflow`` and
+- ``/some/location/my_other_workflow``
+
+respectively:
+
+.. code-block:: shell
+
+   cylc install my_workflow
+   cylc install -C /some/location/my_other_workflow
+
+.. ifnotslides::
+
+   will install your workflow in ``~/cylc-run/my_workflow/runN``.
+
+   (RunN is actually a symlink to runX where X increments each time you
+   install a workflow)
+
+   .. note::
+
+      You can still develop workflows in ``~/cylc-run``, but this is not
+      recommended because development work may
+      change the behaviour of the workflow as it is running.
+
+Validating A Workflow
+---------------------
+
+.. ifnotslides::
+
+   It is a good idea to check a workflow definition for errors before running
+   it. Cylc provides a command which automatically checks the validity of
+   workflow configurations - ``cylc validate``:
 
 .. code-block:: sub
 
    cylc validate <path/to/workflow>
+   cylc validate <workflow_id>  # For workflow in `~/cylc-run/`
 
 .. ifnotslides::
 
    Here ``<path/to/workflow>`` is the path to the workflow's location within the
-   filesystem (so if we create a workflow in ``~/cylc-run/foo`` we would put
-   ``~/cylc-run/foo/flow.cylc``).
+   filesystem (so if we create a workflow in ``~/cylc-src/foo`` we would put
+   ``~/cylc-src/foo/flow.cylc``).
+
+
+Running a workflow
+------------------
 
-   Next we can run the workflow using the ``cylc play`` command.
+.. ifnotslides::
+
+   Now we have installed and validated our workflow we can run the
+   workflow using the ``cylc play`` command.
 
 .. code-block:: sub
 
-   cylc play <name>
+   cylc play <workflow_id>
 
 .. ifnotslides::
 
-   The ``name`` is the name of the :term:`run directory` (i.e. ``<name>``
+   The ``workflow_id`` is the name of the :term:`run directory` (i.e. ``<name>``
    would be ``foo`` in the above example).
 
-.. note::
-
-   In this tutorial we are writing our workflows in the ``cylc-run`` directory.
-
-   However, you should write real workflows in a separate source location
-   (Cylc expects ``~/cylc-src`` by default) and then use ``cylc install`` to
-   install them to the run directory before use. For more information, see
-   :ref:`Installing-workflows`.
 
 Generated Workflow Files
 ------------------------
 
+Numbered run directories
+^^^^^^^^^^^^^^^^^^^^^^^^
+
 .. ifnotslides::
 
-   Cylc generates files and directories when it runs a workflow, namely:
+   .. seealso::
+
+      :ref:`Installing-workflows` for a fuller description of Cylc install,
+      including the option of naming rather than numbering runs.
+
+   By default ``cylc install`` will install your workflow in a new
+   numbered run directory each time you run ``cylc install``:
+
+.. code-block:: sub
```

Comment:
There are no substitutions in this code block.

Use `sub` for:
```
Plain code that includes <substitutions>
```

Use Bash for:
```
echo 'Bash Code'
```

Use `console` for:

```
$ echo 'command line examples'
command line examples
```